paper-tabs.html

```diff
@@ -215,7 +232,7 @@
 
     <paper-icon-button icon="paper-tabs:chevron-left" class$="[[_computeScrollButtonClass(_leftHidden, scrollable, hideScrollButtons)]]" on-up="_onScrollButtonUp" on-down="_onLeftScrollButtonDown" tabindex="-1"></paper-icon-button>
 
-    <div id="tabsContainer" on-track="_scroll" on-down="_down">
+    <div id="tabsContainer" on-track="_scroll" on-wheel="_onWheel" on-down="_down">
```

Comment:
Had to remove this anyways after native scrolling turned out not to be a good option here. :/ I didn't know that about wheel events though.